crates/typst/src/layout/grid/layout.rs

```diff
@@ -376,6 +376,8 @@ impl<'a> GridLayouter<'a> {
 
         self.render_fills_strokes()?;
 
+        self.finished.iter_mut().for_each(|f| f.meta(self.styles, false));
```

Comment:
Could you use a for loop instead?

Reply:
Maybe we could just move this to `finish_region`, right before pushing `output` to `self.finished`? (And maybe add a comment above it explaining the reasoning?)

Reply:
Sure

Reply:
good catch, then not

Reply:
Thanks for verifying!